packaging/windows/packaging/create_install_wizard.iss

```diff
@@ -226,6 +237,35 @@ begin
     Result := True;
 end;
 
+function updateConfigYaml(): Boolean;
+var
+  MappingStrings: TArrayOfString;
+  ConfigYamlPath: String;
+  Status: Boolean;
+  Index: Integer;
+begin
+  ConfigYamlPath := Expandconstant('{app}\Lib\site-packages\cloudify_cli\resources\config.yaml')
+  ExtractTemporaryFile('import_resolver.yaml');
+  Status := LoadStringsFromFile(ExpandConstant('{tmp}\import_resolver.yaml'), MappingStrings);
+
+  if (Status = True) then
```

Comment:
if not Status then
    Result := False;
    Exit;
end

for ...
end;

Result := SaveStringsToFile()